src/main/java/pl/learning/complexOperations/ComplexNumber.java

```diff
@@ -2,6 +2,13 @@
 
 public class ComplexNumber {
 
+    private double real, imaginary;
```

Comment:
Do variables should be over the main method?

Reply:
In that order:

1. Class declaration
2. public fields
3. private fields
4. public methods
5. private methods
i.e:
`public class Person {

    public static final String SOME_GLOBAL_STRING_1 = "stringValue1";
    public static final String SOME_GLOBAL_STRING_2 = "stringValue2";
    public static final String SOME_GLOBAL_STRING_3 = "stringValue3";

    private static final int HEIGHT_IN_METERS_DIVISOR = 100;
    private static final int DEFAULT_BMI_HEIGHT_EXPONENT = 2;

    private int age;
    private int height;
    private String surname;
    private int weight;

    public Person(int age, int height, String surname, int weight) {
        this.age = age;
        this.height = height;
        this.surname = surname;
        this.weight = weight;
    }

    public double getBMI() {
        double heightInMeters = divide(height, Person.HEIGHT_IN_METERS_DIVISOR);
        double heightInPower = power(heightInMeters);

        return divide(weight, heightInPower);
    }

    private double divide(double dividend, double divisor) {
        return dividend / divisor;
    }

    private double power(double base) {
        return Math.pow(base, Person.DEFAULT_BMI_HEIGHT_EXPONENT);
    }
}`